test/unit-test/FreeRTOS_DHCP/FreeRTOS_DHCP_utest.c

```diff
@@ -341,6 +341,12 @@ void test_vDHCPProcess_ResetAndIncorrectStateWithSocketAlreadyCreated( void )
 
         /* Expect these arguments. */
         FreeRTOS_recvfrom_ExpectAndReturn( xDHCPv4Socket, NULL, 0UL, FREERTOS_ZERO_COPY + FREERTOS_MSG_PEEK, NULL, NULL, 0 );
+
+        /* Ignore the source address and source address
+         * length argument though. */
+        FreeRTOS_recvfrom_IgnoreArg_pxSourceAddress();
```

Comment:
Is there any test case verifying multiple DHCP server scenarios?

Reply:
https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/1226/files#diff-33d34c64f4cfe318de92341b204dd23da6d4594abd3a5a81d3a4f360be3a42f8R1347-R1437